src/Tizen.NUI/src/public/Layouting/FlexLayout.cs

```diff
@@ -473,6 +477,8 @@ public FlexWrapType WrapType
             }
             set
             {
+                if (value != FlexWrapType.NoWrap || value != FlexWrapType.Wrap)
```

Comment:
could you add excepton comment here? <exception cref=xxx> xxx </exception>
ex) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69